sdk/servicebus/Azure.Messaging.ServiceBus/MigrationGuide.md

```diff
@@ -212,6 +212,20 @@ while (messages.Count > 0)
 
     // if there are any remaining messages in the .NET queue, the while loop repeats
 }
+
+// create a receiver that we can use to receive the messages
```

Comment:
I don't think we want this change in the migration guide. What we can do instead is define a new region for the receiving part and include both the sending and receiving snippets in the README, but leave only the sending snippet in the migration guide.

Reply:
hi @JoshLove-msft thank you for the comment. I have done as suggested, the migration guide remains untouched.  Introduced a new snippet and included that in the Readme.

Reply:
Excellent - I think all that is remaining is to update the table of contents of the README to reflect the new section and the updated name of the existing section - https://github.com/Azure/azure-sdk-for-net/pull/28221/files#diff-6b20936315df43a04f92031bb5622ac62ef10233db814c3e9aae4f89611bd5c2R145